packages/iris-grid/src/IrisGridIcons.ts

```diff
@@ -18,15 +15,15 @@ export type IconName =
   | 'cellOverflow';
 
 const iconMap = new Map<IconName, IconDefinition>([
-  ['sortUp', dhSortUp],
-  ['sortDown', dhSortDown],
+  ['sortUp', vsTriangleUp],
```

Comment:
Why'd you switch these out? Pretty sure @dsmmcken made the `dhSortUp` and `dhSortDown` icons specifically for the sorting case (and correct sizing).

Reply:
I talked w/ Don about this. He said those icons were probably just to stay consistent w/ the fontawesome icons.

`dhSortUp` and `dhSortDown` are not centered vertically. They are the top/bottom half of the double sort arrows icon. I was trying to center them based on the icon size, but the actual centering was not something clean like shift by `0.25*iconSize`. It had a +/- 1 needed depending on which arrow and I couldn't figure out why or what would make that consistent regardless of icon size.